arch/xtensa/src/esp32/Kconfig

```diff
@@ -324,9 +324,10 @@ config ESP32_I2S1
 		No yet implemented
 
 config ESP32_LEDC
-	bool "LED PWM (LEDC)"
+	bool "LEDC (PWM)"
 	default n
-	depends on EXPERIMENTAL
+	select PWM
+	select ARCH_HAVE_PWM_MULTICHAN
 	---help---
 		No yet implemented
```

Comment:
Probably now it is implemented :)

Reply:
Hahaha, true! I forgot to remove the comment